app/classifier/frame-annotator.jsx

```diff
@@ -110,12 +110,18 @@ export default class FrameAnnotator extends React.Component {
     let BeforeSubject;
     let InsideSubject;
     let AfterSubject;
+    let taskTypeList = [];
     const { type, src } = getSubjectLocation(this.props.subject, this.props.frame);
     const createdViewBox = `${this.props.viewBoxDimensions.x} ${this.props.viewBoxDimensions.y} ${this.props.viewBoxDimensions.width} ${this.props.viewBoxDimensions.height}`;
 
     if (this.props.annotation) {
       taskDescription = this.props.workflow.tasks[this.props.annotation.task];
       TaskComponent = tasks[taskDescription.type];
+      // combo tasks contain many task types
+      taskTypeList = [taskDescription.type];
+      if (taskDescription.type === 'combo') {
+        taskTypeList = taskDescription.tasks.map((task) => { return this.props.workflow.tasks[task].type; });
```

Comment:
The `.eslintrc` currently has `"arrow-body-style": ["warn", "always"]` instead of `as-needed` or `never`, so I assumed we were moving away from the simplified syntax (but I can change it if you want).